ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -1442,14 +1440,15 @@ def test_paddle_log1p(
     )
 
 
-# log2
+# log1p
 @handle_frontend_test(
-    fn_tree="paddle.log2",
+    fn_tree="paddle.log1p",
     dtype_and_x=helpers.dtype_and_values(
         available_dtypes=helpers.get_dtypes("valid"),
+        max_value=1e5,
```

Comment:
don't change other tests

Reply:
I did not, I just pulled the latest changes from the main branch to my branch. You can the see the lint bot did that change in the commit history.

Reply:
Should I close this PR and make another fresh  one because there is so much mess on this one now.

Reply:
🤔 It's marked as a "change" so it is different from what the main branch has. If you can't fix it easily you can open a new PR and close this one, no problem.